.devcontainer/post-install.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/sh
 
-# install WebV
+# install WebV (using the beta for testing)
 # we can't install this in the base image
-dotnet tool install -g webvalidate
+dotnet tool install -g --version 2.0.0-beta1 webvalidate
```

Comment:
Is version 2.0.0-beta1 already available in git hub packages container?

Reply:
Yes, it's on nuget